mycore-base/src/main/java/org/mycore/common/xml/MCRLayoutService.java

```diff
@@ -161,7 +161,7 @@ private String getFileName(HttpServletRequest req, MCRParameterCollector paramet
             return extractFileName(req.getPathInfo());
         }
         return new MessageFormat("{0}-{1}", Locale.ROOT).format(
-            new Object[] { extractFileName(req.getServletPath()), System.currentTimeMillis() });
```

Comment:
The change should work and fix the issue.
But I would remove the MessageFormat completely and simply do a String concatenation or use [String.join()](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-)
`return String.join("-", extractFileName(req.getServletPath(), System.currentTimeMillis());`

Maybe we should replace the other usage of MessageFormat in this class too.